hummingbot/strategy/arbitrage/arbitrage.pyx

```diff
@@ -93,12 +93,12 @@ cdef class ArbitrageStrategy(StrategyBase):
         self.c_add_markets(list(all_markets))
 
     @property
-    def tracked_taker_orders(self) -> List[Tuple[MarketBase, MarketOrder]]:
-        return self._sb_order_tracker.tracked_taker_orders
+    def tracked_maker_orders(self) -> List[Tuple[MarketBase, MarketOrder]]:
+        return self._sb_order_tracker.tracked_maker_orders
```

Comment:
For Arbitrage strategy, there are only taker orders, no maker orders. 
There are currently confusion on `order_tracker.pyx` and `strategy_base.pyx` classes where we take `market` orders as `taker` orders and `limit` orders as makers. They should be considered in insolation of each other actually.

Reply:
What do you suggest?
I thought that modifying the signature of `tracked_taker_orders ` may case a need to modify all files that use it.